users/templates/users/user_login.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div class="row">
+
+        <div class="col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3 col-lg-4 col-lg-offset-4">
+
+            <h2>Login</h2>
+
+            {% if messages %}
+                {% for message in messages %}
```

Comment:
The same template code would have to be written over and over again for every single form. So I thought of creating a template block for this. `include`ing another HTML might be another way.